<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkout</class>
 <widget class="QDialog" name="checkout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{border-width:2px;
border-color:gray;
border-radius:10px;
padding:2px;
border-style:outset;
}
QPushButton:pressed{
background-color: rgb(91, 85, 255);
}

QLineEdit{
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:gray;
padding:3px;
}</string>
  </property>
  <widget class="QPushButton" name="checkOutBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>结账</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancleBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>24</y>
     <width>231</width>
     <height>247</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>   姓 名:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="IdcardLabel">
        <property name="text">
         <string>  身份证:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idCardEdit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="roomIdLabel">
        <property name="text">
         <string>  房间号:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="roomIdEdit"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="startTime">
        <property name="text">
         <string>开始时间:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startTimeEdit"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="endTimeLabel">
        <property name="text">
         <string>结束时间:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="endTimeEdit"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="quitPriceLabel">
        <property name="text">
         <string>退回押金:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="quitPriceEdit_2"/>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="debetLabel">
        <property name="text">
         <string>挂账结余:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="debtEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
